.circleci/config.yml

```diff
@@ -7,22 +7,21 @@ executors:
   default:
     working_directory: ~/timetree-sdk-js/web-api
     docker:
-      - image: circleci/node:12.19.0
+      - image: cimg/node:14.15.1
```

Comment:
Changed to the next generation convenience image.
https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic/